apps/couchdb.go

```diff
@@ -61,3 +62,122 @@ func (r MetricsReceiverCouchdb) Pipelines() []otel.Pipeline {
+					*/
+					Regex: `^\[(?<severity>\w*)\] (?<timestamp>[\d\-\.:TZ]+) (?<node>\w+)@(?<host>[^\s]+) (\w+) [\w-]+ (?<message>[\s\S]*)`,
+					Parser: confgenerator.ParserShared{
+						TimeKey:    "timestamp",
+						TimeFormat: "%Y-%m-%dT%H:%M:%S.%L%z",
+					},
+				},
+				{
+					// Format https://github.com/apache/couchdb/blob/main/src/couch_log/src/couch_log_writer_syslog.erl#L72
+					// Sample line: [info] 2022-01-12T16:52:56.998128Z nonode@nohost <0.216.0> -------- Apache CouchDB has started. Time to relax.
+					Regex: `^\[(?<severity>\w*)\] (?<timestamp>[\d\-\.:TZ]+) (?<node>\w+)@(?<host>[^\s]+) \<(?<pid>[^>]+)\> [\w-]+ (?<message>.*)`,
+					Parser: confgenerator.ParserShared{
+						TimeKey:    "timestamp",
+						TimeFormat: "%Y-%m-%dT%H:%M:%S.%L%z",
+					},
+				},
+			},
+		},
+		Rules: []confgenerator.MultilineRule{
+			{
+				StateName: "start_state",
+				NextState: "cont",
+				Regex:     `^\[\w+\]`,
+			},
+			{
+				StateName: "cont",
+				NextState: "cont",
+				Regex:     `^\[\w+\]`,
+			},
+		},
+	}.Components(tag, uid)
+
+	// Generate the httpRequest structure.
+	c = append(c, fluentbit.Component{
+		Kind: "FILTER",
+		Config: map[string]string{
+			"Name":          "nest",
+			"Match":         tag,
+			"Operation":     "nest",
+			"Wildcard":      "http_request_*",
+			"Nest_under":    "logging.googleapis.com/http_request",
+			"Remove_prefix": "http_request_",
+		},
+	})
+
+	// Log levels documented: https://docs.couchdb.org/en/stable/config/logging.html#log/level
+	c = append(c,
+		fluentbit.TranslationComponents(tag, "level", "logging.googleapis.com/severity", false,
+			[]struct{ SrcVal, DestVal string }{
+				{"emerg", "EMERGENCY"},
+				{"emergency", "EMERGENCY"},
+				{"alert", "ALERT"},
+				{"crit", "CRITICAL"},
+				{"critical", "CRITICAL"},
+				{"error", "ERROR"},
+				{"err", "ERROR"},
+				{"warn", "WARNING"},
+				{"warning", "WARNING"},
+				{"notice", "NOTICE"},
+				{"info", "INFO"},
+				{"debug", "DEBUG"},
+			},
+		)...,
+	)
+	return c
+}
+
+type LoggingReceiverCouchdbError struct {
+	LoggingProcessorCouchdbError            `yaml:",inline"`
+	confgenerator.LoggingReceiverFilesMixin `yaml:",inline" validate:"structonly"`
+}
+
+func (r LoggingReceiverCouchdbError) Components(tag string) []fluentbit.Component {
+	if len(r.IncludePaths) == 0 {
+		r.IncludePaths = []string{
+			// Default log file
+			"/var/log/couchdb/couchdb.log",
+		}
+	}
+	c := r.LoggingReceiverFilesMixin.Components(tag)
+	c = append(c, r.LoggingProcessorCouchdbError.Components(tag, "couchdb_error")...)
```

Comment:
fix to be "couchdb" like the type named above.

Reply:
fixed